policies/aws_cloudtrail_policies/aws_cloudtrail_s3_bucket_public.py

```diff
@@ -7,7 +7,49 @@
 }
 
 
+# https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-ct-placeholder-buckets.html
+PLACEHOLDER_BUCKETS = {
```

Comment:
Would it make sense to have a set of `EXCLUDED_BUCKETS` that additional bucket names can be added to?

Also slight nit, can these names be sorted alphabetically?